src/components/forms/EmailField/EmailField.js

```diff
@@ -0,0 +1,44 @@
+import {
+  FormFieldDescription,
+  FormLabel,
+  Paragraph,
+  Textbox,
+  FormField as UtrechtFormField,
+} from '@utrecht/component-library-react';
+import {Field} from 'formik';
+import React from 'react';
+
+import {getBEMClassName} from 'utils';
+
+export const EmailField = ({
+  name = '',
+  label = '',
+  isRequired = false,
+  description = '',
+  id = '',
+  disabled = false,
+  invalid = false,
+}) => {
+  const labelClassName = getBEMClassName('label', [isRequired && 'required'].filter(Boolean));
+
+  const inputProps = {
+    name,
+    as: Textbox,
+    id,
+    disabled,
+    invalid,
+    type: 'email', // Set the input type to 'email'
+  };
+
+  return (
+    <UtrechtFormField type={'text'} invalid={invalid}>
+      <Paragraph className={labelClassName}>
+        <FormLabel htmlFor={id}>{label}</FormLabel>
+      </Paragraph>
+      <Paragraph>
+        <Field name={name} as={Textbox} {...inputProps} />
+      </Paragraph>
+      {description && <FormFieldDescription invalid={invalid}>{description}</FormFieldDescription>}
+    </UtrechtFormField>
+  );
+};
```

Comment:
When comparing this against the code of the `TextField` component, this is 90% the same - that's also not so crazy, since it's essentially just an input with `type="email"` set.

So, for nicer code organization, I'd actually implement this as a form of `TextField`:

```jsx
import TextField from '../TextField';

const EmailField = (props) => <TextField {...props} type="email" />

const {type, ...emailPropTypes} = TextField.propTypes;
EmailField.propTypes = emailPropTypes;

export default EmailField;
```